airflow/config_templates/default_airflow.cfg

```diff
@@ -176,6 +176,9 @@ store_serialized_dags = False
 # Updating serialized DAG can not be faster than a minimum interval to reduce database write rate.
 min_serialized_dag_update_interval = 30
 
+# On each dagrun check against defined SLAs
```

Comment:
Can you add some documentation about this option?   If there is no information about this feature in the documentation, very few people will be able to use it.

Reply:
@mik-laj - > where do you think the documentation should be added ? Maybe you can provide some pointers? Most of the config options are documented in comments of the default_airlfow.cfg. Any other specific place you think this documentation should be added (and where all the previous options are documented?)

Reply:
> and where all the previous options are documented?
We have documentation gaps, but we try to develop documentation all the time. There is no precise documentation for the SLA mechanism, but this does not mean that it is impossible to write a few sentences and put them in the documentation. Another person who wants to describe the wider documentation will have less work.

I think that these two sentences describing this parameter may be placed in a new document in the howto directory. I will give an example of documentation that describes the configuration parameter.
https://github.com/apache/airflow/pull/5850/files

Reply:
Created at https://github.com/apache/airflow/pull/6974

Reply:
@kaxil Fantastic!

Reply:
Great! Thanks @mik-laj and @kaxil for explaining and adding :).